api/db-cdr/changelog/db.changelog-master.xml

```diff
@@ -10,4 +10,7 @@
   </preConditions>
```

Comment:
Did you decide not to add indexes for now?

Reply:
sorry.. i will add them